metrics/internal/cache/pv.go

```diff
@@ -358,3 +401,108 @@ func CreatePersistentVolumeListWatch(kubeClient clientset.Interface, fieldSelect
 		},
 	}
 }
+
+func runCephFSSubvolumeCount(config *cephMonitorConfig, cephFilesystemName string) (map[string]int, error) {
+
+	groups, err := runCephFSSubvolumeGroups(config, cephFilesystemName)
+	if err != nil {
+		return nil, fmt.Errorf("failed to get subvolumegroups: %w", err)
+	}
+
+	result := make(map[string]int)
+	for _, groupName := range groups {
+		count, err := runCephFSSubvolumeCountPerGroup(config, cephFilesystemName, groupName)
```

Comment:
IN case of provider or the external mode, are we able to run this one as user might be restricted per SVG?